src/components/tag/tag/tag.ts

```diff
@@ -37,6 +40,22 @@ export class SbbTagElement extends SbbIconNameMixin(
   /** Whether the tag is checked. */
   @property({ reflect: true, type: Boolean }) public checked = false;
 
+  /** Tag size. */
+  @property({ reflect: true })
+  public set size(value: SbbTagSize) {
+    this._size = value;
+  }
+  public get size(): SbbTagSize {
+    return this.group?.size ?? this._size;
+  }
+  private _size: SbbTagSize = 'm';
+
+  /** Reference to the connected checkbox group. */
+  public get group(): SbbTagGroupElement | null {
+    return this._group;
+  }
```

Comment:
Although it's not wrong, I think we don't need to make it public. What was your thought?
```suggestion
```

Reply:
I didn't have any particular thought, I just copied the implementation from the radios/checkboxes. Should we change it there as well?

Reply:
Hmm I think there they were really used, but here they aren't. But still, it is not wrong :). @kyubisation ?